distribution/docs/src/main/jdocs/content/_configuring/hardening-intro-contents.adoc

```diff
@@ -0,0 +1,15 @@
+:title: Hardening Intro
+:type: configuringIntro
+:status: published
+:summary: Introduction to Hardening.
+:order: 03
+
+.Security Hardening
+****
```

Comment:
I was going to start using it more, but am now finding that it doesn't translate well to pdf if there are page breaks.